api-samples/Sandbox/sandbox/main.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Listens for the extension launching then creates the window
+ *
+ * @see http://developer.chrome.com/apps/app.runtime.html
+ * @see http://developer.chrome.com/apps/app.window.html
+ */
+chrome.runtime.onStartup.addListener(() => {
```

Comment:
Can we do chrome.runtime.onInstalled? Otherwise the developer will have to restart the browser to try this, which is a bit frustrating.